.tekton/release.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: cachi2-on-release
+  annotations:
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-event: "[push]"
+    pipelinesascode.tekton.dev/on-target-branch: "[refs/tags/*]"
+    pipelinesascode.tekton.dev/task: "[git-clone]"
+spec:
+  params:
+    - name: repo_url
+      value: "{{repo_url}}"
+    - name: revision
+      value: "{{revision}}"
+  pipelineSpec:
+    tasks:
+      - name: fetch-repository
+        taskRef:
+          name: git-clone
+        workspaces:
+          - name: output
+            workspace: workspace
+        params:
+          - name: depth
+            value: "0"
+          - name: url
+            value: $(params.repo_url)
+          - name: revision
+            value: $(params.revision)
+      - name: release
+        runAfter:
+          - fetch-repository
+        workspaces:
+          - name: output
+            workspace: workspace
+        taskSpec:
+          results:
+            - name: version
+          workspaces:
+            - name: output
+          steps:
+            - name: get-semver-tag
+              image: registry.access.redhat.com/ubi9/python-39
+              workingDir: $(workspaces.output.path)
+              env:
+              - name: WORKSPACE_OUTPUT_PATH
+                value: $(workspaces.output.path)
+              script: |
+                #!/usr/bin/env bash
+                set -eufx
+                
+                git config --global --add safe.directory "${WORKSPACE_OUTPUT_PATH}"
+                git fetch --tag -v
+                version=$(git  --no-pager tag --points-at HEAD)
+                [[ -z ${version} ]] && {
+                    echo "No tags detected"
+                    exit 1
```

Comment:
nitpick: maybe "No tag points at this commit" or "No tag points at commit $PARAM_REVISION"?

Otherwise, the "No tags detected" error message would follow a list of tags printed by git fetch --tag -v :slightly_smiling_face: